src/components/ErrorBoundary/BaseErrorBoundary.tsx

```diff
@@ -1,32 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import {ErrorBoundary} from 'react-error-boundary';
 import BootSplash from '../../libs/BootSplash';
 import GenericErrorPage from '../../pages/ErrorPage/GenericErrorPage';
 
-const propTypes = {
+type LogError = (message: string, error: Error, errorInfo: string) => void;
+
+type BaseErrorBoundaryProps = {
```

Comment:
We should define this common type in types.ts file

Reply:
moved